### ISSUE TITLE
"Match color" item background too light, makes some text hard to read.

### ISSUE THREAD

**Issue by stamminator** [AdrianWilczynski/OneDarkPro#16]
![image](https://user-images.githubusercontent.com/10633170/112515070-52cbbb80-8d6c-11eb-9d5a-795c207b78f4.png)

When changing the "Match color" display item background to a darker color, it makes text much more readable without having too little contrast to quickly spot the matches.

![image](https://user-images.githubusercontent.com/10633170/112515579-c1a91480-8d6c-11eb-9fc9-0c2cee7ad48f.png)

If it's possible to use outlines instead of background colors for match colors like Visual Studio Code does, that would be ideal...

![image](https://user-images.githubusercontent.com/10633170/112519028-582b0500-8d70-11eb-897f-f77a61ec7f91.png)

...but if not, a darker background color should suffice. I used RGB(81, 41, 163), which is just a darker shade of the existing Default color. Tested it on several different symbols in different languages and it looks pretty good. I'd suggest bringing such a change into the project.

**Comment by flobernd** [comment]
I've choosen `RGB(54, 59, 69)` so that the `Match Text` color matches the one inside of the outline box:

![screenshot](https://nextcloud.flobernd.de/s/6dTrk3mPrDfDXjJ/preview)

I think that should be the way it's done in the standard themes as well.

**Comment by icymaru** [comment]
Yeah it's really hard to see the highlight especially the yellow one.
![image](https://user-images.githubusercontent.com/6220818/199917395-c4be532f-c8ed-40ed-954a-24cea7791182.png)
